dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/AppsHelper.java

```diff
@@ -380,50 +380,50 @@ private void validateIncomingParams(final Set<String> incomingParamNames, final
      * @throws IOException
      * @throws DotDataException
      */
-    List<ServiceIntegrationView> createServiceIntegration(final FormDataMultiPart multipart, final User user)
+    List<AppView> createApp(final FormDataMultiPart multipart, final User user)
             throws IOException, DotDataException {
         final List<File> files = new MultiPartUtils().getBinariesFromMultipart(multipart);
         if(!UtilMethods.isSet(files)){
             throw new DotDataException("Unable to extract any files from multi-part request.");
         }
-        List<ServiceIntegrationView> serviceIntegrationViews = new ArrayList<>(files.size());
+        List<AppView> appViews = new ArrayList<>(files.size());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'appViews' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)